### DISCUSSION TITLE
zh/article/prophet-civilization-history/%E4%B8%BA%E4%BB%80%E4%B9%88%E8%A6%81%E5%86%99%E5%85%88%E7%9F%A5-%E6%96%87%E6%98%8E%E5%8F%B2

### DISCUSSION THREAD

**Discussion by giscus** [vuepress-theme-hope/giscus-discussions#315] [Announcements]
# zh/article/prophet-civilization-history/%E4%B8%BA%E4%BB%80%E4%B9%88%E8%A6%81%E5%86%99%E5%85%88%E7%9F%A5-%E6%96%87%E6%98%8E%E5%8F%B2

专注于学习、分享、交流

http://localhost:8080/zh/article/prophet-civilization-history/%E4%B8%BA%E4%BB%80%E4%B9%88%E8%A6%81%E5%86%99%E5%85%88%E7%9F%A5-%E6%96%87%E6%98%8E%E5%8F%B2.html

**Comment by StormWangxhu** [comment]
很不错
